algebird-test/src/test/scala/com/twitter/algebird/MinPlusLaws.scala

```diff
@@ -17,16 +17,9 @@ limitations under the License.
 package com.twitter.algebird
 
 import org.scalatest.{Matchers, PropSpec}
-import org.scalatest.prop.PropertyChecks
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
```

Comment:
I guess this must also be unused since the file body didn’t change but we can fix it when we error on warnings and put unused deps.